ajax/widget/js/widget.js

```diff
@@ -0,0 +1,40 @@
+let employeeList = new XMLHttpRequest();
+
+employeeList.onreadystatechange = () => {
+    if (employeeList.readyState === 4) {return;}
```

Comment:
I know I did the sample with this on one line, but I think most people prefer to see the return in its own block as in not inlined.
Also I had changed the conditional to a strict not equals.

Reply:
Done